cli/tests/pcluster/aws/dummy_aws_api.py

```diff
@@ -265,6 +265,13 @@ def describe_storage_virtual_machines(self, storage_virtual_machine_ids):
             result.append({"Endpoints": {"Nfs": {"DNSName": "abcd"}}})
         return result
 
+    def describe_file_caches(self, file_cache_ids):
+        """Describe FSx File cache."""
+        result = []
+        for _ in file_cache_ids:
+            result.append({"DNSName": "abcd", "LustreConfiguration": {"MountName": "/abcd"}})
```

Comment:
*Minor: Code Style*
When settings mocked/testing values it's a good practice to not suggest relations between values that does not exists in the real case. In this specific line you are whispering to the developer that there exists a relation between the DNS name and the mount name, but this is not the case. I suggest to use something like "a.dns.name" and "/amoutname" instead of "abcd" and "/abcd". This is a good practice also because it make it easier to read the debugger outputs in case we need to troubleshoot with it.

Reply:
Got it. I have been following the pattern that other Storage mockers have set.